wasm-wrappers/src/lib.rs

```diff
@@ -513,6 +516,50 @@ pub fn encode_output_create_stake_pool(
     Ok(output.encode())
 }
 
+/// Returns the fee that needs to be paid by a transaction for issuing a new fungible token
```

Comment:
Please also document the required parameters. Remember that the documentation doesn't include that.